### PR TITLE
Refactor for more editor support

### PR DESCRIPTION
**Pull request by ajnieset**
Changed the logic to allow for many editors

### CODE REVIEW

**Comment by Rexogamer** [comment]
Hmmm... looks good, but I feel problems *may* arise.

**Comment by Rexogamer** [comment]
note to reviewer: I made some edits and have forked Austin’s fork so he’ll have to merge those first

**Comment by ajnieset** [comment]
What problems do feel may arise?

**Comment by Rexogamer** [comment]
If the editor’s command is different from most I guess... although in that case I presume we’d add an exception.

Also, I opened a PR on your fork

**Comment by ajnieset** [comment]
yea thats kind of why i added the try except block so that it errors out on wrong/not found commands for editors. But the setup config should be setup properly by the user, so there is some reliance that the user knows a little bit on what they are doing. Ive skimmed your PR and it i do like the documentation to help with editor support.

**Comment by jarodburchill** [comment]
Austin, if you like the changes Ed made, merge them into this pull request so I can see everything in one place! Thanks guys!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ProjectAutomation
 Python script for creating new projects in the desired local directory, with a GitHub origin.
 ## Requirements:
-- Python 3.x
+- Python 3.6+
```
Comment (jarodburchill) [inline README.md:4]: Good call.

File: automate-project.py [hunk 2]
```diff
@@ -133,17 +133,16 @@ def CreateGitHubRepo():
...
-    print("No editor selected.")
-else
-    print(bcolors.WARNING + "Editor unknown. Please consult config.script."
-print(bcolors.OKGREEN + "Project created succesfully!" + bcolors.ENDC)
-
+editor = config.get("DEFAULT", "editor")
 
+if editor is not "none":
+    try:
+        subprocess.run(f"{editor} .", shell=True)
```
Comment (jarodburchill) [inline automate-project.py:140]: This is much cleaner now! Awesome!

**Review by jarodburchill — APPROVED** [review]
Awesome stuff! Closes #10

**Pull request merged**